src/utils/apiUrls.js

```diff
@@ -0,0 +1,6 @@
+const apiUrls = {
+    domainName: 'https://api.themoviedb.org/3/',
+    posterUrl: 'https://image.tmdb.org/t/p/w220_and_h330_face/',
+    apiKey:'7c4a986b16b3b892bd7111a358d63e05'
+}
+export default apiUrls;
```

Comment:
```suggestion
const domainName = 'https://api.themoviedb.org/3/',
const posterUrl = 'https://image.tmdb.org/t/p/w220_and_h330_face/',
const apiKey ='7c4a986b16b3b892bd7111a358d63e05'
export {
    domainName,
    posterUrl,
    apiKey
};
```

To use it on a file:

```suggestion
import {domainName} from '<path-to>/utils/apiUrls';
```

Reply:
I have read about the differences between `import` and `import default`, Now I have a good grasp about how those work. Thank you :1st_place_medal: